### PR TITLE
Centering elements

### PR DESCRIPTION
**Pull request by alex0popa**

### CODE REVIEW

File: codewell-fe/src/Code/Add.js [hunk 1]
```diff
@@ -48,23 +48,30 @@ function AddCode() {
...
-          onChange={onCodeChange}
-          name="UNIQUE_ID_OF_DIV"
-          editorProps={{ $blockScrolling: true }}
-          value={code}
-        />,
-        <Button type="primary" id="submit-button" onClick={submitCode}>
-          {t('Submit')}
-        </Button>
-      </Row>
+      <Layout style={{ marginLeft: '20%', marginRight: '20%' }}>
```
Comment (S7012MY) [inline codewell-fe/src/Code/Add.js:51]: why exactly 20% here? What does this do?

File: codewell-fe/src/Code/Add.js [hunk 2]
```diff
@@ -48,23 +48,30 @@ function AddCode() {
...
+        <Space direction="vertical">
+          <Row>
+            <ChooseLanguage onLanguageChange={onLanguageChange}/>
+          </Row>
+          <Layout style={{ border: '1px solid #bfbfbf' }}>
```
Comment (S7012MY) [inline codewell-fe/src/Code/Add.js:56]: can we extract this to a variable?

File: codewell-fe/src/Code/Add.js (hunk 1 above)
Comment (alex0popa) [reply codewell-fe/src/Code/Add.js:51]: I wanted to have the editor in the center of the page and responsive, but I realized that I could include it in a layout and put it in the center.

File: codewell-fe/src/Code/Add.js (hunk 2 above)
Comment (alex0popa) [reply codewell-fe/src/Code/Add.js:56]: of course :D

File: codewell-fe/src/User/Register.js [hunk 3]
```diff
@@ -21,109 +21,112 @@ function Register() {
...
-        >
-          <Form.Item label={t('Username')} name="id"
-            rules={[
-              {
-                required: true,
-                message: t('Please input your username!'),
-              },
-            ]}
+    <Layout style={{ alignItems: 'center' }}>
+      <Card style={{ width: 400, border: '1px solid #bfbfbf'  }}>
```
[the remark this answers is not in the document]
Comment (alex0popa) [reply codewell-fe/src/User/Register.js:25]: "why exactly 400?" -> I wanted it to be wider than the default width of the card

"Why do we need borders?" ->  Mihai's suggestion

File: codewell-fe/src/User/Login.js [hunk 4]
```diff
@@ -24,41 +24,47 @@ function Login(props) {
...
-          name="login"
-          className="login-form"
-          onFinish={onFinish}
-        >
-          <Form.Item
-            name="id"
-            rules={[{ required: true,
-              message: t('Please input your username!') }]}
+      <Layout style={{ alignItems: 'center' }}>
+        <Card style={{ width: 300, border: '1px solid #bfbfbf' }}>
```
[the remark this answers is not in the document]
Comment (alex0popa) [reply codewell-fe/src/User/Login.js:28]: "why 300" -> I had set the width before including it in a layout and I didn't realize it was no longer needed.

"why borders?" ->  Mihai's suggestion

**Review by S7012MY — APPROVED** [review]

**Pull request merged**